awx/ui/client/src/templates/job_templates/multi-credential/multi-credential-modal.directive.js

```diff
@@ -111,7 +111,7 @@ function multiCredentialModalController(GetBasePath, qs, MultiCredentialService)
 
         scope.credentialTypes.forEach((credentialType => {
             if(credentialType.kind
-                .match(/^(machine|cloud|net|ssh|vault)$/)) {
+                .match(/^(machine|cloud|net|ssh|vault|kubernetes)$/)) {
```

Comment:
We'll also need to make sure the new category is displayed on launch prompts in `ui_next`:
![cred_prompt](https://user-images.githubusercontent.com/9753817/87680207-1fe81700-c74b-11ea-9d8b-bfaf71cd3448.gif)

That code lives [here](https://github.com/ansible/awx/blob/devel/awx/ui_next/src/components/LaunchPrompt/steps/CredentialsStep.jsx#L119)